runtime/parachains/src/configuration/migration.rs

```diff
@@ -51,7 +48,7 @@ pub mod v3 {
 		pub validation_upgrade_delay: BlockNumber,
 		pub max_pov_size: u32,
 		pub max_downward_message_size: u32,
-		pub ump_service_total_weight: OldWeight,
+		pub ump_service_total_weight: Weight,
```

Comment:
I'm a bit confused. Why is this change here? Kusama's configuration is on version 3, Polkadot is still on version 2. We are not we planning to skip v3 for Polkadot, are we? Even if we are, this migration is incorrect for Kusama.

Reply:
Ah, sorry, I misinterpreted the diff. This looks correct, but we need to make sure Polkadot upgrades to v3 before putting a migration for v4.

Reply:
But if Polkadot and Kusama are not at the same version  - we have got a problem? I shouldn't wipe out the migration to v3?

Reply:
See https://github.com/paritytech/polkadot/pull/6345#discussion_r1059966155.

Reply:
I understand now. Thanks @ordian!